server/api/assessment/getReviewStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express'
+
+import { CountryIso } from '@meta/area'
+
+import { AssessmentController } from '@server/controller/assessment'
+import { CycleDataController } from '@server/controller/cycleData'
+import Requests from '@server/utils/requests'
+
+export const getReviewStatus = async (req: Request, res: Response) => {
+  try {
+    const { countryIso, assessmentName, cycleName, section } = <Record<string, string>>req.query
+
+    const { assessment, cycle } = await AssessmentController.getOneWithCycle({ name: assessmentName, cycleName })
+
+    const returnedOriginalDataPoint = await CycleDataController.getReviewStatus({
```

Comment:
returnedOriginalDataPoint might come from copy and paste ;) call it reviewStatus

Reply:
O_O